extensions/functions_boolean.yaml

```diff
@@ -1,14 +1,6 @@
 %YAML 1.2
 ---
 scalar_functions:
-  -
-    name: not
```

Comment:
Should we keep the function but correct it to have a single argument?

Reply:
as pointed out to me, apparently we already had the correct one elsewhere in the file.